src/commands/TruffleCommands.ts

```diff
@@ -100,6 +100,20 @@ export namespace TruffleCommands {
     Telemetry.sendEvent("TruffleCommands.deployContracts.commandFinished");
   }
 
+  export async function execScript(uri: Uri): Promise<void> {
+    Telemetry.sendEvent("TruffleCommands.execScript.commandStarted");
+
+    await showIgnorableNotification(Constants.statusBarMessages.executingScript, async () => {
+      if (!(await required.checkAppsSilent(RequiredApps.truffle))) {
+        Telemetry.sendEvent("TruffleCommands.execScript.truffleInstallation");
+        await required.installTruffle(required.Scope.locally);
+      }
+      await outputCommandHelper.executeCommand(getWorkspaceRoot(), "npx", RequiredApps.truffle, "exec", uri.fsPath);
```

Comment:
do you not want to do something with the output of this call? 

Does this output to the outputChannel or similar? Debugging any issues will be problematic otherwise. Especially any exceptions (non 0 return codes).

Reply:
hey @michaeljohnbennett (apols for the delay), yup it does currently output to `Truffle for VSCode` channel...does that make sense from your perspective?

Reply:
@kevinbluer, just a heads up:  if we have more than one Truffle Project opened, or even a root directory without any truffle-config.js file, the function getWorkspaceRoot() might not work (e.g.: deploy function). The fix for that is on: fix/compile-from-working-dir